netbox/data_source_netbox_vlan.go

```diff
@@ -18,21 +18,25 @@ func dataSourceNetboxVlan() *schema.Resource {
 			"vid": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				AtLeastOneOf: []string{"name", "vid"},
 				ValidateFunc: validation.IntBetween(1, 4094),
 			},
 			"name": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				AtLeastOneOf: []string{"name", "vid"},
+				Type:     schema.TypeString,
+				Optional: true,
 			},
 			"description": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"group": {
```

Comment:
Can you rename `group` to `group_id`? I know this is inconsistent in this particular data source, but generally, I prefer when fields with IDs have the `_id` suffix. Sometimes this slips through the review.

I will sometime write a design document about this for easier reference.

You can leave the old fields as is to preserve backwards compatibility. I will eventually deprecated them and replace them with `_id` fields.

Reply:
Renamed.

Is there an easy way to make aliases to fields in order to have e.g. `group` and `group_id` fields, which act identically? I'm wondering this, because it might be useful to support old and new name of a field for a while when renaming fields.

Reply:
I (re-)added a deprecated field in https://github.com/e-breuninger/terraform-provider-netbox/commit/be7fad02cd977b44f6f4b6855c3e51099469a84e

While this is not an "alias" field, it is simple enough to implement quickly IMO.